### PR TITLE
Add Firehose sink + rename namespace of existing Kinesis sink

### PR DESCRIPTION
**Pull request by endofcake**
Now that Amazon has Kinesis Streams, Kinesis Firehose, and Kinesis
Analytics, we need to separate the Streams functionality into a new
namespace so that we could later add Firehose, while reusing some of
the shared finctionality.

Some functionality is exactly the same for both Firehose and Streams.
Makes sense to have it in a common namespace.

Kinesis Firehose is essentially copy-pasted from Kinesis Streams sink,
could certainly use some refactoring love. Sometime.

@thirkcircus

### CODE REVIEW

File: src/Serilog.Sinks.Amazon.Kinesis.Firehose/packages.config [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="AWSSDK" version="2.3.52.0" targetFramework="net452" requireReinstallation="true" />
```
Comment (thirkcircus) [inline src/Serilog.Sinks.Amazon.Kinesis.Firehose/packages.config]: this one looks like it ought not to be here...

Comment (endofcake) [inline src/Serilog.Sinks.Amazon.Kinesis.Firehose/packages.config]: @thirkcircus , fixed

**Comment by endofcake** [comment]
@thirkcircus , renamed the namespace.

**Comment by endofcake** [comment]
@thirkcircus , by the way it's building now.

**Pull request merged**